app.py

```diff
@@ -5,6 +5,10 @@
 import json, os
 from requests import get
 
+from lib.func import serve, start_agent, hydra_doc, apidoc_graph, send_command, default
```

Comment:
When you have renamed `func.py` to `utils.py`, I think you need to fix the imports here from `lib.func` to `lib.utils`.